src/lib/helpers/getFetchClient.ts

```diff
@@ -38,27 +38,25 @@ export const getFetchClient = (konfigStore: Store): {
     }
 
     return (input: FetchInputParameter, init?: FetchInitParameterWithClient) =>
-        fetchShim(konfigStore, input, init);
+        fetchShim(config, input, init);
 };
 
 function fetchShim(
-    konfigStore: Store,
+    config: Config,
     input: FetchInputParameter,
     init?: FetchInitParameterWithClient,
 ): FetchReturn {
-    const fetchTimeout = konfigStore.get("networking.fetch_timeout_ms");
-    const fetchRetry = konfigStore.get("networking.fetch_retry_enable");
-    const fetchMaxAttempts = konfigStore.get("networking.fetch_retry_times");
-    const fetchInitialDebounce = konfigStore.get(
-        "networking.fetch_retry_initial_debounce",
-    );
-    const fetchDebounceMultiplier = konfigStore.get(
-        "networking.fetch_retry_debounce_multiplier",
-    );
+    const fetchTimeout = config.networking.fetch?.timeout_ms;
+    const fetchRetry = config.networking.fetch?.retry?.enabled;
+    const fetchMaxAttempts = config.networking.fetch?.retry?.times;
```

Comment:
this is how the optional typing comes out. It's pretty natural with the optional chaining in JS